### PR TITLE
feat: implement initials name

### PR DESCRIPTION
**Pull request by geovie**
Implements initials name function

### CODE REVIEW

File: thaw/src/avatar/mod.rs [hunk 1]
```diff
@@ -144,3 +152,16 @@ impl AvatarShape {
...
+
+#[test]
+fn test_initials_name() {
+    assert_eq!(initials_name("Jane Doe".into()), "JD".to_string());
+    assert_eq!(initials_name("Ben".into()), "B".to_string());
+    assert_eq!(
+        initials_name("ÇFoo Bar 1Name too ÉLong".into()),
+        "ÇÉ".to_string()
+    );
+    assert_eq!(initials_name("ﬄ ß".into()), "FFLSS".to_string());
```
Comment (geovie) [inline thaw/src/avatar/mod.rs:164]: This could also be changed to return `FS` if wanted

Comment (luoxiaozero) [reply thaw/src/avatar/mod.rs:164]: Yes. The string length should not exceed 2.

Comment (geovie) [reply thaw/src/avatar/mod.rs]: Changed 👍

**Comment by luoxiaozero** [comment]
Thanks!

**Pull request merged**